### DISCUSSION TITLE
ErrorException error while creating new project

### DISCUSSION THREAD

**Discussion by BinitaShakya** [pimcore/pimcore#10237] [Q&A / Support]
I am trying to install pimcore 10 using these commands
`COMPOSER_MEMORY_LIMIT=-1 composer create-project pimcore/skeleton my-project`

I am getting this error
![image](https://user-images.githubusercontent.com/6542919/131994270-e988adc3-4994-47bc-a0ba-112f191a9bb4.png)

I am using windows10, xampp, php version is 8.0.10 and composer 2.0.8
Can anybody please help?

**Comment by dvesh3** [comment]
We are reviewing this PR https://github.com/pimcore/pimcore/pull/10201 to fix the issue.

**Reply by BinitaShakya** [reply]
when can I expect this to be merge?

**Reply by dvesh3** [reply]
We are not sure about the fix and this not on our list for the moment.
